include/nanogui/glutil.h

```diff
@@ -465,6 +465,8 @@ struct Arcball {
 
     Quaternionf &state() { return mQuat; }
 
+    Quaternionf getState() const { return mQuat; }
```

Comment:
Please use the same naming conventions:
``const Quaternionf &state() const { return mQuat; }``

If you chose this different name to avoid an error in the python bindings, you'll need to use ``py::overload_cast<>(&Quaternionf::state)``